src/rabbit.erl

```diff
@@ -745,6 +752,16 @@ is_running() -> is_running(node()).
 
 is_running(Node) -> rabbit_nodes:is_process_running(Node, rabbit).
 
+is_booted(Node) ->
+    case rpc:call(Node, rabbit, is_booted, []) of
```

Comment:
Technically this won't be available on pre-3.7.4 nodes. But given that vhosts are not created during rolling upgrades in practice, this might be OK.

Reply:
Right. We can check for processes directly instead.